src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/ScriptActionsOnRunningCluster.cs

```diff
@@ -122,7 +122,7 @@ public void TestScriptActionsOnRunningCluster()
                     var result = client.Clusters.ExecuteScriptActions(resourceGroup, dnsName, executeScriptActionParams);
                     Assert.Equal(result.StatusCode, HttpStatusCode.OK);
                     Assert.Equal(result.State, AsyncOperationState.Failed);
-                    Assert.Equal(result.ErrorInfo.Message, "ScriptExecutionFailed");
```

Comment:
I removed the line on the recommendation of the test owner. She is working to fix the underlying issue.